titus_isolate/exit_handler.py

```diff
@@ -0,0 +1,8 @@
+from abc import abstractmethod
+
+
+class ExitHandler:
```

Comment:
can we get rid of this abstraction? seems unnecessary

Reply:
It's for tests.  It was hard to write a test that for cpu allocator changes which cause a system exit, especially since we do "hard" exit.